src/app/pages/swap/swap.context.ts

```diff
@@ -8,6 +8,7 @@ export interface SwapSubmissionData extends SwapFormValues {
   liquidityFee: number;
   protocol: string;
   router: SwapAsset[];
+  dexPath: string[];
```

Comment:
Mmm, sorry see here this isn't necessarily Bitflow specific? Alex has this too?

Reply:
The type specifically does follow the Bitflow SDK convention (string array), but having a DEX path in general would be generic for any DEX aggregator.

I updated the component that displays this `<SwapDetails>` to be aware that its optional.